std-bits/table/src/main/java/org/enso/table/operations/AddRunning.java

```diff
@@ -0,0 +1,134 @@
+package org.enso.table.operations;
+
+import org.enso.base.statistics.Statistic;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.column.storage.numeric.DoubleStorage;
+import org.enso.table.data.table.Column;
+import org.enso.table.problems.ProblemAggregator;
+
+public class AddRunning {
+
+  public static Storage<Double> create_running(
+      Statistic statistic,
+      Column sourceColumn,
+      Column[] groupingColumns,
+      Column[] orderingColumns,
+      int[] directions,
+      ProblemAggregator problemAggregator) {
+    if (orderingColumns.length != directions.length) {
+      throw new IllegalArgumentException(
+          "The number of ordering columns and directions must be the same.");
+    }
+    var runningGenerator =
+        RunningGenerator.createGenerator(
+            sourceColumn, groupingColumns, orderingColumns, directions, problemAggregator);
+    runningGenerator.generate(new RunningIteratorFactoryImpl(statistic));
+    var ret =
+        new DoubleStorage(
+            runningGenerator.result, sourceColumn.getSize(), runningGenerator.isNothing);
```

Comment:
Running Count should be a LongStorage
And Min/Max should be same type as the input I think?

Reply:
RunningCount is still using the Add_Row_Number code path so works.

Agree Min/Max (and Sum?) should match the input type. Do you want me to do that now or merge this one first?

Reply:
Will do in next MR